src/components/content/home/user/ServicesDashboard.tsx

```diff
@@ -5,11 +5,11 @@
 
 import React from 'react';
 import { Card, Col, Row, Statistic } from 'antd';
-import { ServiceVo } from '../../../xpanse-api/generated';
-import useListDeployedServicesQuery from '../myServices/query/useListDeployedServicesQuery';
+import { ServiceVo } from '../../../../xpanse-api/generated';
+import useListDeployedServicesQuery from '../../myServices/query/useListDeployedServicesQuery';
 import { ServicesDashboardSkeleton } from './ServicesDashboardSkeleton';
 import ServicesDashboardError from './ServicesDashboardError';
-import { myServicesRoute } from '../../utils/constants';
+import { myServicesRoute } from '../../../utils/constants';
 import { createSearchParams, useNavigate } from 'react-router-dom';
 import serviceDeploymentState = ServiceVo.serviceDeploymentState;
 
```

Comment:
All the end user dashboard related components can start with -> EndUserServicesDashboard